espn_api/football/league.py

```diff
@@ -65,8 +65,11 @@ def _fetch_draft(self):
             round_pick = pick['roundPickNumber']
             bid_amount = pick['bidAmount']
             keeper_status = pick['keeper']
-
-            self.draft.append(Pick(team, playerId, playerName, round_num, round_pick, bid_amount, keeper_status))
+            if self.get_team_data(pick['nominatingTeamID']) != 0:

```

Comment:
No need for this check, in `self.get_team_data` it will return `None` if not found.

Field name should be `nominatingTeamId`